thoth/storages/sync.py

```diff
@@ -464,92 +465,107 @@ def sync_dependency_monkey_documents(
-    from amun import has_inspection_job
-    from amun import is_inspection_finished
-
-    if is_local:
-        raise NotImplementedError("Cannot sync inspection documents from a local file")
-
-    if only_graph_sync and only_ceph_sync:
-        raise ValueError("At least one of Ceph or Graph should be performed")
-
-    inspection_store = InspectionResultsStore()
-    inspection_store.connect()
-
-    dependency_monkey_reports_store = DependencyMonkeyReportsStore()
-    dependency_monkey_reports_store.connect()
+    if is_local and not document_ids:
+        raise ValueError(
+            "Cannot sync documents from local directory without explicitly specifying a list of documents to be synced"
+        )
 
     if not graph:
         graph = GraphDatabase()
         graph.connect()
 
     processed, synced, skipped, failed = 0, 0, 0, 0
-    for inspection_id in document_ids or dependency_monkey_reports_store.iterate_inspection_ids():
-        processed += 1
-        if force or not inspection_store.document_exists(inspection_id):
-            finished = is_inspection_finished(amun_api_url, inspection_id)
+    for inspection_document_id in document_ids or InspectionStore.iter_inspections():
 
-            if finished:
-                _LOGGER.info("Obtaining results from Amun API for inspection %r", inspection_id)
-                try:
-                    specification, created = get_inspection_specification(amun_api_url, inspection_id)
-                    build_log = get_inspection_build_log(amun_api_url, inspection_id)
-                    status = get_inspection_status(amun_api_url, inspection_id)
-                    job_log = None
-                    if has_inspection_job(amun_api_url, inspection_id):
-                        job_log = get_inspection_job_log(amun_api_url, inspection_id)
-
-                    document = {
-                        "specification": specification,
-                        "created": created,
-                        "build_log": build_log,
-                        "job_log": job_log,
-                        "inspection_id": inspection_id,
-                        "status": status,
-                    }
-
-                    # First we store results into graph database and then onto
-                    # Ceph. This way in the next run we can sync documents that
-                    # failed to sync to graph - see if statement that is asking
-                    # for Ceph document presents first.
-                    if not only_ceph_sync:
-                        _LOGGER.info(f"Syncing inspection {inspection_id!r} to graph")
-                        graph.sync_inspection_result(document)
-
-                    if not only_graph_sync:
-                        _LOGGER.info(f"Syncing inspection {inspection_id!r} to {inspection_store.ceph.host}")
-                        inspection_store.store_document(document)
+        if inspection_document_id.startswith("inspection"):
```

Comment:
Why is this check needed?

Reply:
maybe it was just in that bucket I'm using, as there are files which are not starting with `inspection`

```
                           PRE amun-inspection-workflow/
                           PRE inspection-test-04af8c87/
                           PRE inspection-test-0617b28e/
                           PRE inspection-test-123d9e0e/
                           PRE inspection-test-1267aeef/
                           PRE inspection-test-169240ce/
                           PRE inspection-test-1f8e4ee6/
                           PRE inspection-test-233da921/
                           PRE inspection-test-24090f9d/
                           PRE inspection-test-298a78e5/
                           PRE inspection-test-33cf7cdf/
                           PRE inspection-test-3441c9aa/
                           PRE inspection-test-468c3a90/
                           PRE inspection-test-4d7f8549/
                           PRE inspection-test-5dd6a866/
                           PRE inspection-test-6a806054/
                           PRE inspection-test-6a993395/
                           PRE inspection-test-6ca0f000/
                           PRE inspection-test-74c72eae/
                           PRE inspection-test-7cd1e56e/
                           PRE inspection-test-800da468/
                           PRE inspection-test-8068028d/
                           PRE inspection-test-80f107d7/
                           PRE inspection-test-82b9dcc8/
                           PRE inspection-test-837f59b3/
                           PRE inspection-test-8d63e3e0/
                           PRE inspection-test-9013a114/
                           PRE inspection-test-966b0ba6/
                           PRE inspection-test-9a0ea348/
                           PRE inspection-test-9f8fbb5a/
                           PRE inspection-test-a068bd81/
                           PRE inspection-test-ad71e98d/
                           PRE inspection-test-b3533477/
                           PRE inspection-test-b8d1e9c9/
                           PRE inspection-test-b9ca4dcd/
                           PRE inspection-test-bed25a64/
                           PRE inspection-test-c13b4c8c/
                           PRE inspection-test-e34b8ae7/
                           PRE results/
```

Reply:
Oh, the `results` is misplaced. You can expect the inspection to be always present in the correct structure. So no need for the check.